chore: update srvx section (#58)
paoloricciuti/tmcp: packages/transport-sse/README.md

### `srvx`

If you want to have the same experience throughout Deno, Bun or Node you can also use [`srvx`](https://srvx.h3.dev/).

```js

> import { McpServer } from 'tmcp";

Author: ```suggestion
import { McpServer } from 'tmcp';
```

### `srvx`

> If you want to have the same experience throughout Deno, Bun or Node you can also use [`srvx`](https://srvx.h3.dev/).

Author: ```suggestion
If you want the same experience across Deno, Bun, and Node.js, you can use [srvx](https://srvx.h3.dev/).
```
